tests/bluetooth/tester/overlay-mesh-v1d1.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_ENTROPY_GENERATOR=y
+
+CONFIG_BT_MESH_V1d1=y
+CONFIG_BT_MESH_OP_AGG_CLI=y
+CONFIG_BT_MESH_OP_AGG_SRV=y
+# PTS requires more key slots.
+# First one is implicitly taken by Device Key.
+CONFIG_BT_MESH_MODEL_KEY_COUNT=3
+CONFIG_BT_MESH_LARGE_COMP_DATA_CLI=y
+CONFIG_BT_MESH_LARGE_COMP_DATA_SRV=y
+CONFIG_BT_MESH_SAR_CFG_SRV=y
+CONFIG_BT_MESH_SAR_CFG_CLI=y
+CONFIG_BT_MESH_TX_SEG_MSG_COUNT=10
+CONFIG_BT_MESH_PROVISIONER=y
```

Comment:
It has been already set here: https://github.com/PavelVPV/zephyr/blob/mesh_1_1_tester/tests/bluetooth/tester/overlay-mesh.conf#L20